Makefile

```diff
@@ -320,7 +320,7 @@ lint: ## (re)run the linter
 fmt: $(BUILD)/fmt ## run goimports
 
 # not identical to the intermediate target, but does provide the same codegen (or more).
-copyright: ## update copyright headers
+copyright: $(BIN)/copyright ## update copyright headers
```

Comment:
oops.  I must've forgotten this one.  sorry!

Reply:
did it cause the "syntax error: unexpected word (expecting ")")"?

Reply:
I don't think so. The error message seemed to be from something make was running, not make itself